src/pybind/mgr/progress/module.py

```diff
@@ -194,21 +195,31 @@ def __init__(self, message, refs, add_to_ceph_s, start_epoch, active_clean_num):
         self._active_clean_num = active_clean_num
         self._refresh()
 
-    def global_event_update_progress(self, pg_dump):
-        # type: (Dict) -> None
+    def global_event_update_progress(self, pg_dump, log):
+        # type: (Dict, Any) -> None
         "Update progress of Global Recovery Event"
 
         pgs = pg_dump['pg_stats']
         new_active_clean_num = 0
-        for pg in pgs:
+        skipped_pgs = 0
 
-            if int(pg['reported_epoch']) < int(self._start_epoch):
-                continue
+        for pg in pgs:
+            # Disregard PGs that are not being reported
+            # if the states are active+clean. Since it is
+            # possible that some pgs might not have any movement
+            # even before the start of the event.
 
             state = pg['state']
 
             states = state.split("+")
 
+            if int(pg['reported_epoch']) < int(self._start_epoch):
```

Comment:
why do we need to cast these variables to `int`?

Reply:
you're right, the types of both are already int